subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dependencies/DefaultMinimalDependency.java

```diff
@@ -21,60 +21,15 @@
 
 import java.io.Serializable;
 
-public class DefaultMinimalDependency implements MinimalExternalModuleDependency, Serializable {
-    private final ModuleIdentifier module;
-    private final MutableVersionConstraint versionConstraint;
-    private final int hashCode;
-
+public class DefaultMinimalDependency extends DefaultExternalModuleDependency implements MinimalExternalModuleDependency, Serializable {
```

Comment:
And this class declaration reads even odder.

Reply:
I may be able to modify these internal names, but I don't know if that would make much sense if the public types keep their names.